tests/all.js

```diff
@@ -1033,11 +1124,13 @@ test('/index fallback + unsee', function() {
 
   equal(count, 3);
 
-  deepEqual(require._stats, {
+  var stats = statsForMonitor('loaderjs', tree);
+
+  deepEqual(stats, {
     findDeps: 3,
     define: 2,
     exports: 3,
-    findModule: 6,
+    findModule: 7,
```

Comment:
I would assert in between each and at the beginning to see where the count goes wrong, that will likely give us what we need to dig into which part of the tree / which part of loader is doing something suspicious.